### PR TITLE
write up Director responsibilities

### LINKED ISSUE #227: write up Director responsibilities
Part of #262.

I have not done a good enough job clarifying what I see as being responsibilities for the Director of the Technology Portfolio specifically, versus members of the team generally.

## TODOs

- [x] Review the [position description](https://join.tts.gsa.gov/join/technology-portfolio-director/) for ideas
- [x] Write up a list

### PR DESCRIPTION
**Pull request by afeld**
Replacement for #323. Closes #227. Builds on #322.

### CODE REVIEW

File: roles.md [hunk 1]
```diff
@@ -14,5 +14,27 @@
...
+  - [Leave approvals](https://handbook.18f.gov/leave/)
+  - [Performance management](https://handbook.18f.gov/performance-management/)
+- Solutions leadership responsibilities, such as:
+  - Attending Senior Management Team (SMT) meetings
+- Ensuring the team members feel supported as individuals
+- Ensuring the team is supported by other leadership
+- Ensuring the team fulfills its various [responsibilities](https://handbook.tts.gsa.gov/tech-portfolio/), including meeting deadlines
+- Resolving conflicts within the team, including being a tie-breaker when needed
+
+The following are _not_ responsibilities of the Director specifically, in contrast to managers in other teams/organizations:
```
Comment (its-a-lisa) [inline roles.md]: I would recommend saying that these responsibilities of the Director are morphed from what the typical manager is; phrasing it more in a positive than a negative.

File: roles.md [hunk 2]
```diff
@@ -14,5 +14,27 @@
...
+
+- Focusing on strategy
```
Comment (its-a-lisa) [inline roles.md]: Dictating strategy in a vacuum

File: roles.md [hunk 3]
```diff
@@ -14,5 +14,27 @@
...
+- Setting vision/goals/deadlines
```
Comment (its-a-lisa) [inline roles.md]: Having a shared vision rather than a sole Director's vision

File: roles.md [hunk 4]
```diff
@@ -14,5 +14,27 @@
...
+- Assigning tasks
```
Comment (its-a-lisa) [inline roles.md]: Prefer self organization as opposed to strict delegation

File: roles.md [hunk 5]
```diff
@@ -14,5 +14,27 @@
   - Authorizing Official
   - Backup
 - [Infrastructure-as-a-Service (IaaS)](https://before-you-ship.18f.gov/infrastructure/) administrator
-- Tech Portfolio Specialist / all team members
+- [Tech Portfolio Specialist](https://join.tts.gsa.gov/join/technology-portfolio-specialist/) / all team members
 - TTS SaaS PM
+
+## Director
```
Comment (its-a-lisa) [inline roles.md]: When you've spoken about your vision for the Director role and the TTS Tech Portfolio team; you've mentioned 'flat' organization and 'command and control' -- I recommend including those ideas _somewhere_ if not here.

Comment (afeld) [reply roles.md]: I suggest we put that in "how we work." cc https://github.com/18F/tts-tech-portfolio/issues/248

Comment (afeld) [reply roles.md]: On second thought, including these helped clarify why the bottom list is in there.

**Pull request merged**